Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -660,18 +711,25 @@ private static InputOverlayDrawableButton initializeOverlayButton(Context contex
 	/**
 	 * Initializes an {@link InputOverlayDrawableDpad}
 	 *
-	 * @param context     The current {@link Context}.
-	 * @param resId       The resource ID of the {@link Drawable} to get the {@link Bitmap} of.
-	 * @param buttonUp    Identifier for the up button.
-	 * @param buttonDown  Identifier for the down button.
-	 * @param buttonLeft  Identifier for the left button.
-	 * @param buttonRight Identifier for the right button.
+	 * @param context                     The current {@link Context}.
+	 * @param defaultResId                The {@link Bitmap} resource ID of the default sate.
+	 * @param pressedOneDirectionResId    The {@link Bitmap} resource ID of the pressed sate in one direction.
+	 * @param pressedTwoDirectionsResId   The {@link Bitmap} resource ID of the pressed sate in two directions.
+	 * @param buttonUp                    Identifier for the up button.
+	 * @param buttonDown                  Identifier for the down button.
+	 * @param buttonLeft                  Identifier for the left button.
+	 * @param buttonRight                 Identifier for the right button.
 	 *
 	 * @return the initialized {@link InputOverlayDrawableDpad}
 	 */
-	private static InputOverlayDrawableDpad initializeOverlayDpad(Context context, int resId,
-								      int buttonUp, int buttonDown,
-								      int buttonLeft, int buttonRight)
+	private static InputOverlayDrawableDpad initializeOverlayDpad(Context context,
+																  int defaultResId,
+																  int pressedOneDirectionResId,
+																  int pressedTwoDirectionsResId,
+																  int buttonUp,
+																  int buttonDown,
+																  int buttonLeft,
+																  int buttonRight)
```

Comment:
If you can't fix it in Studio, open it in a basic text editor (like Notepad, if you use windows). It should display like it will on GitHub, and you'll be able to align it properly.